<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectVideoFun</class>
 <widget class="QWidget" name="SelectVideoFun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>805</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SelectVideoFun</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择视频文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deal">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>处理视频</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>941</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>处理结果</string>
    </attribute>
    <widget class="QPushButton" name="show">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>显  示</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>401</width>
       <height>391</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>399</width>
        <height>389</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>处理前</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>处理后</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>170</y>
       <width>400</width>
       <height>391</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>398</width>
        <height>389</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>统计明细</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>871</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="showde">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>显示分析明细</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1111</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/TrafficCon/image/back3.jpg);</string>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>deal</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>deal</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>show</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>scrollArea_2</tabstop>
 </tabstops>
 <resources>
  <include location="TrafficCon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectVideoFun</receiver>
   <slot>getVideoPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deal</sender>
   <signal>clicked()</signal>
   <receiver>SelectVideoFun</receiver>
   <slot>dealVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show</sender>
   <signal>clicked()</signal>
   <receiver>SelectVideoFun</receiver>
   <slot>showPic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showde</sender>
   <signal>clicked()</signal>
   <receiver>SelectVideoFun</receiver>
   <slot>showDetail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>1077</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getVideoPath()</slot>
  <slot>dealVideo()</slot>
  <slot>showPic()</slot>
  <slot>showDetail()</slot>
 </slots>
</ui>
